src/domain/agent/credential/OfferAlkemioCommunityCredentialDialog.tsx

```
@@ -124,9 +124,9 @@ const OfferAlkemioCommunityCredentialDialog: FC<OfferAlkemioCommunityCredentialD
         )}
         <Button
           onClick={async () => {
-            if (selectedContribution && selectedContribution.communityId) {
+            if (selectedContribution && selectedContribution.roleSetId) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)